### PR TITLE
Turn menu creation methods into extension methods

### PR DESCRIPTION
**Pull request by shana**
We're going to need to create menus from more places.

Also consolidate extensions some more, and remove unused menu command id

### CODE REVIEW

File: src/GitHub.Exports/Extensions/VSExtensions.cs [hunk 1]
```diff
@@ -75,5 +81,64 @@ static object GetServiceAndCache<CacheType>(IServiceProvider provider, Type type
...
+                    ((OleMenuCommand)s).Visible = canEnable();
+                },
+                id);
+            mcs.AddCommand(item);
+        }
+    }
+
+    public static class ISolutionExtensions
+    {
+        public static UriString GetRepoUrlFromSolution(IVsSolution solution)
```
Comment (Haacked) [inline src/GitHub.Exports/Extensions/VSExtensions.cs]: This is not an extension method. Is that intentional?

**Comment by Haacked** [comment]
:peach: This looks good! Just one minor question.

File: src/GitHub.Exports/Extensions/VSExtensions.cs (hunk 1 above)
Comment (shana) [inline src/GitHub.Exports/Extensions/VSExtensions.cs]: It's an oversight. This method isn't being currently used, and the way it stands right now, it doesn't have valuable logic that's worth keeping it around (GetRepoFromSolution does the same exact thing), so might just kill it.

Comment (Haacked) [inline src/GitHub.Exports/Extensions/VSExtensions.cs]: :fire: it!

**Comment by shana** [comment]
:tada:

**Comment by Haacked** [comment]
Excellente!

**Pull request merged**